src/django_pg_migration_tools/operations.py

```diff
@@ -281,11 +319,13 @@ def _show_lock_timeout(
         self,
         schema_editor: base_schema.BaseDatabaseSchemaEditor,
     ) -> str:
-        cursor = schema_editor.connection.cursor()
-        cursor.execute(TimeoutQueries.SHOW_LOCK_TIMEOUT)
-        result = cursor.fetchone()[0]
-        assert isinstance(result, str)
-        return result
+        return _run_introspection_query(
+            schema_editor,
+            psycopg_sql.SQL(TimeoutQueries.SHOW_LOCK_TIMEOUT).as_string(
+                schema_editor.connection.connection
+            ),
```

Comment:
It might be a good idea to move all of them for consistency sake
I've opened a separated issue here to explore that idea -> https://github.com/kraken-tech/django-pg-migration-tools/issues/58